packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/index.hbs

```diff
@@ -25,19 +25,24 @@
           <CardPay::BalanceDisplay
             class="face-value-card__selected-amount-display"
             @size="large"
-            @icon={{this.faceValueToken.icon}}
+            @icon="spend"
+            @sign="§"
+            @symbol="SPEND"
             @balance={{this.selectedFaceValue}}
-            @usdBalance={{mult this.selectedFaceValue this.spendToUSDRate}}
-            @text={{concat @amountInFundingToken " " this.fundingToken.symbol "*"}}
+            @usdBalance={{mult this.selectedFaceValue this.spendToUsdRate}}
+            @text={{concat "≈ " (mult this.selectedFaceValue this.spendToUsdRate) " " this.fundingToken.symbol "*"}}
             data-test-face-value-display
           />
         </CardPay::LabeledValue>
       {{else}}
         <div class="face-value-card__options">
-          {{#each this.amountOptions as |amount|}}
+          {{#each this.faceValueOptions as |amount|}}
             <CardPay::IssuePrepaidCardWorkflow::FaceValue::Option
+              @disabled={{lt this.balanceFromBN (mult amount this.spendToUsdRate)}}
               @checked={{eq amount this.selectedFaceValue}}
-              @spendAmount={{amount}}
+              @spendFaceValue={{amount}}
+              @usdAmount={{mult amount this.spendToUsdRate}}
+              @approxDaiAmount={{mult amount this.spendToUsdRate}}
```

Comment:
Approximate DAI amount can probably use the `convertFromSpend` method from the SDK?